inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/ClickHouseService.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.audit.service;
+
+import org.apache.inlong.audit.config.ClickHouseConfig;
+import org.apache.inlong.audit.db.entities.ClickHouseDataPo;
+import org.apache.inlong.audit.protocol.AuditData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.concurrent.Executors;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * ClickHouseService
+ */
+public class ClickHouseService implements InsertData, AutoCloseable {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ClickHouseService.class);
+    public static final String INSERT_SQL = "insert into audit_data (ip, docker_id, thread_id,\r\n"
+            + "      sdk_ts, packet_id, log_ts,\r\n"
+            + "      inlong_group_id, inlong_stream_id, audit_id,\r\n"
+            + "      count, size, delay, \r\n"
+            + "      update_time)\r\n"
+            + "    values (?,?,?,?,?,?,?,?,?,?,?,?,?)";
+
+    private ClickHouseConfig chConfig;
+
+    private ScheduledExecutorService timerService = Executors.newScheduledThreadPool(1);
+    private LinkedBlockingQueue<ClickHouseDataPo> batchQueue;
+    private AtomicBoolean needBatchOutput = new AtomicBoolean(false);
+    private AtomicInteger batchCounter = new AtomicInteger(0);
+
+    private Connection conn;
+
+    /**
+     * Constructor
+     * @param chConfig
```

Comment:
It is suggested to add some description for param, return and throw in Java docs.
Otherwise, IDEA or other IDE will be warned.

Reply:
[add method parameter description.](https://github.com/apache/inlong/pull/4747/commits/14e8f2b599452ec0b53dd4285975993eee366331)